source/apiVolontaria/apiVolontaria/tests/tests_view_Users.py

```diff
@@ -537,6 +536,13 @@ def test_list_users(self):
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_view_users_with_filter(self):
+        self.client.force_authenticate(user=self.admin)
+
+        response = self.client.get(reverse('users') + "?search=%s" % self.user.email)
+
+        self.assertEqual(json.loads(response.content)['count'], 1)
+
```

Comment:
This test is good for me, just make sure to create other test for other field accepted in the research.

Reply:
Okay, added